.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,94 @@
+name: Publish Docker images
+
+on:
+  push:
+    branches:
+      - edge
+  release:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  # packages: write
+
+jobs:
+  build-container-image:
+    name: Build Docker image ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: Debian
+            file: Docker/Dockerfile
+            flavor: |
+              latest=auto
+            tags: |
+              type=edge
+              type=semver,pattern={{version}}
+              type=semver,pattern={{major}}.{{minor}}
+              type=semver,pattern={{major}},enable=${{ !startsWith(github.ref, 'refs/tags/0.') }}
+          - name: Alpine
+            file: Docker/Dockerfile-Alpine
+            flavor: |
+              latest=false
+            tags: |
+              type=raw,value=alpine,enable=${{ github.ref == 'refs/heads/latest' || startsWith(github.ref, 'refs/tags/') }}
+              type=edge,suffix=-alpine,onlatest=false
+              type=semver,pattern={{version}}-alpine
+              type=semver,pattern={{major}}.{{minor}}-alpine
+              type=semver,pattern={{major}}-alpine,enable=${{ !startsWith(github.ref, 'refs/tags/0.') }}
+    steps:
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Get FreshRSS version
+      run: |
+        FRESHRSS_VERSION=$(sed -n "s/^const FRESHRSS_VERSION = '\(.*\)'.*$/\1/p" constants.php)
+        echo "$FRESHRSS_VERSION"
+        echo "FRESHRSS_VERSION=$FRESHRSS_VERSION" >> $GITHUB_ENV
+
+    - name: Add metadata to Docker images
+      id: meta
+      uses: docker/metadata-action@v5
+      with:
+        flavor: ${{ matrix.flavor }}
+        images: |
+          docker.io/freshrss/freshrss
+      #     ghcr.io/${{ github.repository }}
+        tags: ${{ matrix.tags }}
+        labels: |
+          org.opencontainers.image.revision=${{ github.ref_name }}.${{ github.sha }}
+          org.opencontainers.image.version=${{ env.FRESHRSS_VERSION }}
```

Comment:
this is already handle by the `org.opencontainers.image.revision` label which (the commit hash)
dont know if there is benefit to know which branch is build from when you have the commit

also thinks the container version label is fine with sometthing like `1.22.2` or `1.22.2-alpine`its specify the container version not the version of the application itself

Reply:
> if there is benefit to know which branch is build from

This is mostly because this is what we already have, and I find it a bit easier to read with something like `1.22.1.cc2878aed855463efe36cf6fb96d001dbb75612e`

> also thinks the container version label is fine with sometthing like 1.22.2 or 1.22.2-alpineits specify the container version not the version of the application itself

I would also be happy with `1.22.2-alpine`, but that does not work for e.g. `edge` builds

Reply:
Switched back to default revision https://github.com/FreshRSS/FreshRSS/pull/5808/commits/3d110b8ec9af4ba9423890c1e018e99c8fbbaec1